packages/lesswrong/components/notifications/SubscribeTo.jsx

```diff
@@ -43,7 +43,7 @@ const SubscribeTo = ({
       view: "subscriptionState",
       documentId: document._id,
       userId: currentUser?._id,
-      subscriptionType,
+      type: subscriptionType,
       collectionName,
```

Comment:
Note: I think this was previously a bug, which we didn't run into because thus far most subscriptions haven't relied on manually passing in a "type" to the SubscribeTo component